spark-scala/src/test/scala/com/matthewrathbone/spark/SparkJoinsTest.scala

```diff
@@ -0,0 +1,36 @@
+package test.scala.com.matthewrathbone.spark
+
+import org.scalatest.junit.AssertionsForJUnit
+import org.junit.Before
+import org.apache.spark.SparkConf
+import org.apache.spark.SparkContext
+import org.junit.Test
+import scala.collection.Map
+import org.apache.spark.SparkContext._
+
+class SparkJoinsTest extends AssertionsForJUnit {
+  
+  var sc: SparkContext = _
+  
+  @Before def initialize() {
+	 val conf = new SparkConf().setAppName("SparkJoins").setMaster("local")
+	 sc = new SparkContext(conf)
+  }
+  
+  @Test def sortByKey() {
+    val transactions: List[Tuple2[Int, Int]] = List(new Tuple2[Int, Int](1, 1), new Tuple2[Int, Int](2, 1),
```

Comment:
There's a short-hand for making a tuple, you just put them in parens. No need to declare the class and all the types.

```scala
val tuple = (1, 1)
val other = new Tuple2[Int, Int](1, 1)
tuple == other // true
```